.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+## Intent
+
+> Describe what problem this pull request resolves, or what new feature it
+> introduces.
+>
+> - Link to related GitHub issues, if applicable
```

Comment:
If you use HTML-style comments, then these notes won't render as part of the description.

Reply:
I kinda like optimizing for the editing view, though, since that's where folks will interact with these most of the time.